integration-tests/gradle/build.gradle.kts

```diff
@@ -0,0 +1,5 @@
+tasks.wrapper {
```

Comment:
Can you add a comment that this (and `settings.gradle.kts` are rather "dummy" files?

Reply:
since it's already merged should I create separate PR?

Reply:
Yes, please @grossws. Thanks.

Reply:
Ok, will create new PR soon